docs/pip.md

```diff
@@ -484,6 +485,34 @@ hermeto-output/deps/pip
 └── ...
 ```
 
+## Working with Rust-based dependencies
+
+Hermeto provides a way to prepare hermetic build of a Python package which depends on
+Rust-based packages. This process cannot be fully automated on Hermeto's side, thus it would
+require minor intervention from users.
+
+Building such project requires that all build dependencies are  listed in
+requirements-build.txt. To achieve this
+[pybuild-deps](https://pypi.org/project/pybuild-deps) could be used. The
+following command will take care of generating the list:
+
+```
+pybuild-deps compile --generate-hashes -o requirements-build.txt requirements.txt
+```
+
+Note, that this step requires internet connection.
+
+Once requirements-build.txt is populated fetch could be done as usual. Hermeto will fetch all
+run time and build time dependencies for both Python and Rust parts.
+
+Note, that a system which is to be used for building these extensions must have
+both `rustc` and `cargo` installed. In some cases it might be necessary to preinstall
+C libraries which the Rust packages need to be built.
```

Comment:
I'd probably reword this to a less verbose version of
_must have `rustc`, `cargo`, and all necessary C libraries in order to build the Rust extension packages installed._